src/azul/deployment.py

```diff
@@ -280,6 +281,32 @@ def resource(self, *args, **kwargs):
 aws = AWS()
 
 
+class Terraform(object):
```

Comment:
```suggestion
class Terraform:
```

Reply:
I didn't realize that all classes are new-style by default in Python 3. Should this be added to the contributing guide?

Reply:
I don't think so. I'd file that under common sense.